Lib/fontmake/font_project.py

```diff
@@ -62,15 +64,44 @@ def __init__(self, timing=False, verbose='INFO'):
             configLogger(logger=timer.logger, level=logging.DEBUG)
 
     @timer()
-    def build_master_ufos(self, glyphs_path, family_name=None):
+    def build_master_ufos(self, glyphs_path, family_name=None, mti_source=None):
         """Build UFOs and MutatorMath designspace from Glyphs source."""
         import glyphsLib
-
         master_dir = self._output_dir('ufo')
         instance_dir = self._output_dir('ufo', is_instance=True)
-        return glyphsLib.build_masters(
+        _, designspace_path, instances = glyphsLib.build_masters(
             glyphs_path, master_dir, designspace_instance_dir=instance_dir,
             family_name=family_name)
+        if mti_source:
+            self.add_mti_features_to_master_ufos(mti_source, designspace_path)
+        return designspace_path, instances
+
+    @timer()
+    def add_mti_features_to_master_ufos(self, mti_source, designspace_path):
+        with open(mti_source, 'rb') as mti_file:
+            mti_paths = readPlist(mti_file)
+        mti_dir = os.path.dirname(mti_source)
+        master_dir = os.path.dirname(designspace_path)
+        designspace = etree.parse(designspace_path)
+        for master in designspace.findall('sources/source'):
+            master_path = os.path.join(master_dir, master.attrib['filename'])
+            data_dir = os.path.join(master_path, 'data')
+            if not os.path.exists(data_dir):
+                os.mkdir(data_dir)
+            target_dir = os.path.join(
+                data_dir, 'com.github.googlei18n.ufo2ft.mtiFeatures')
+            os.mkdir(target_dir)
```

Comment:
would this `os.mkdir(target_dir)` raise `FileExistsError` (or `OSError` 17) if the `target_dir` already exists?